test/foundry-tests/integration/RelayOrderReactorIntegration.t.sol

```diff
@@ -125,30 +122,45 @@ contract RelayOrderReactorIntegrationTest is GasSnapshot, Test, Interop, PermitS
         vm.revertTo(snapshot);
     }
 
-    // swapper creates one order containing a universal router swap for 100 DAI -> USDC
-    // order contains two inputs: DAI for the swap and USDC as gas payment for fillers
-    // at the forked block, 95276229 is the minAmountOut
-    function testExecute() public {
```

Comment:
Can we keep testExecute? And just make this a new function?

Reply:
that way we still have the old testExecute benchmark

Reply:
Yep sounds good